packages/ember-views/tests/views/view_test.js

```diff
@@ -41,31 +41,9 @@ QUnit.test('should not add role attribute unless one is specified', function() {
   ok(view.$().attr('role') === undefined, 'does not have a role attribute');
 });
 
-QUnit.test('should allow tagName to be a computed property [DEPRECATED]', function() {
-  view = EmberView.extend({
-    tagName: computed(function() {
-      return 'span';
-    })
-  }).create();
-
-  expectDeprecation(function() {
-    run(function() {
-      view.createElement();
-    });
-  }, /using a computed property to define tagName will not be permitted/);
-
-  equal(view.element.tagName, 'SPAN', 'the view has was created with the correct element');
-
-  run(function() {
-    view.set('tagName', 'div');
-  });
-
-  equal(view.element.tagName, 'SPAN', 'the tagName cannot be changed after initial render');
-});
-
 import { test } from 'internal-test-helpers/tests/skip-if-glimmer';
 
-test('should re-render if the context is changed', function() {
+QUnit.skip('should re-render if the context is changed', function() {
```

Comment:
This is specifically testing context and we are removing the implementation of context in this PR.